models/node_helpers.go

```diff
@@ -60,23 +60,23 @@ func checkUniqueNodeName(q *reform.Querier, name string) error {
 	}
 }
 
-func checkUniqueNodeInstanceRegion(q *reform.Querier, instance, region string) error {
+func CheckUniqueNodeInstanceRegion(q *reform.Querier, instance, region string) (*Node, error) {
 	if instance == "" {
-		return status.Error(codes.InvalidArgument, "Empty Node instance.")
+		return nil, status.Error(codes.InvalidArgument, "Empty Node instance.")
```

Comment:
A lot of API tests (with https://github.com/Percona-Lab/pmm-api-tests/pull/70/files checked out) fail with this error message.

Reply:
With the up-to-date branch all tests are passing except:
```
--- FAIL: TestAuth (0.13s)
    --- PASS: TestAuth/AuthErrors (0.00s)
        --- PASS: TestAuth/AuthErrors//401 (0.01s)
            auth_test.go:44: URI: https://127.0.0.1:8443/v1/version
        --- PASS: TestAuth/AuthErrors/bad:wrong/401 (0.03s)
            auth_test.go:44: URI: https://bad:wrong@127.0.0.1:8443/v1/version
    --- PASS: TestAuth/NormalErrors (0.00s)
        --- PASS: TestAuth/NormalErrors/Unauthenticated/401 (0.03s)
        --- PASS: TestAuth/NormalErrors/PermissionDenied/403 (0.03s)
    --- FAIL: TestAuth/Setup (0.06s)
        --- PASS: TestAuth/Setup/Redirect (0.00s)
            --- PASS: TestAuth/Setup/Redirect/qan=303 (0.00s)
            --- PASS: TestAuth/Setup/Redirect/swagger=303 (0.00s)
            --- PASS: TestAuth/Setup/Redirect/v1/AWSInstanceCheck=405 (0.01s)
            --- PASS: TestAuth/Setup/Redirect/prometheus=303 (0.01s)
            --- PASS: TestAuth/Setup/Redirect/graph=303 (0.01s)
            --- PASS: TestAuth/Setup/Redirect/v1/version=401 (0.02s)
            --- PASS: TestAuth/Setup/Redirect/v1/readyz=200 (0.06s)
        --- PASS: TestAuth/Setup/API (0.00s)
        --- FAIL: TestAuth/Setup/WebPage (0.00s)
            auth_test.go:105: 
                        Error Trace:    auth_test.go:105
                        Error:          Not equal: 
                                        expected: 200
                                        actual  : 401
                        Test:           TestAuth/Setup/WebPage
                        Messages:       response:
                                        {"code":16,"error":"Unauthorized","message":"Unauthorized"}
            auth_test.go:106: 
                        Error Trace:    auth_test.go:106
                        Error:          Should be true
                        Test:           TestAuth/Setup/WebPage
                        Messages:       {"code":16,"error":"Unauthorized","message":"Unauthorized"}
```